pkg/storage/storage.go

```diff
@@ -177,6 +185,10 @@ func GetPlatformStorage(listers *regopclient.Listers) (imageregistryv1.ImageRegi
 	// EmptyDir storage.
 	case configapiv1.LibvirtPlatformType:
 		fallthrough
+	case configapiv1.AlibabaCloudPlatformType:
```

Comment:
This makes the operator use OSS on `libvirt`. Please put this `case` before the case for `LibvirtPlatformType`.

Reply:
done